src/main/java/org/skriptlang/skript/bukkit/damagesource/elements/CondScalesWithDifficulty.java

```diff
@@ -0,0 +1,63 @@
+package org.skriptlang.skript.bukkit.damagesource.elements;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.damage.DamageSource;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.bukkit.damagesource.DamageSourceExperiment;
+
+@Name("Damage Source - Does Scale With Difficulty")
+@Description("Whether the damage from a damage source scales with the difficulty of the server.")
+@Example("""
+	on death:
+		if event-damage source scales with difficulty:
+	""")
+@Since("INSERT VERSION")
+@RequiredPlugins("Minecraft 1.20.4+")
+public class CondScalesWithDifficulty extends PropertyCondition<DamageSource> implements DamageSourceExperiment {
+
+	static {
+		Skript.registerCondition(CondScalesWithDifficulty.class,
+			"%damagesources% (does scale|scales) with difficulty",
+			"%damagesources% (does not|doesn't) scale with difficulty");
```

Comment:
```suggestion
			"%damagesources% (does scale|scales) damage with difficulty",
			"%damagesources% (does not|doesn't) scale damage with difficulty");
```
this might be pedantic but the source isn't scaling with difficulty, the damage is